.github/workflows/release.yml

```diff
@@ -24,3 +26,68 @@ jobs:
           goarch: ${{ matrix.goarch }}
           ldflags: "-s -w -X main.version=${{ github.ref_name }}"
           binary_name: "gql-lint"
+
+  lipo:
+    name: Universal Mac binary
+    runs-on: macos-12
+    needs: releases-matrix
```

Comment:
I tried to keep it as a separated workflow that was triggered after this was completed (on workflow_run), but it didn't work and I was losing my patience, so, I made it a a different job in this workflow

Reply:
makes sense to keep it together imo 👍